cmd/dagger/module.go

```diff
@@ -1020,6 +1218,10 @@ func (m *moduleDef) GetObjectFunction(objectName, functionName string) (*modFunc
 }
 
 func (m *moduleDef) GetFunction(fp functionProvider, functionName string) (*modFunction, error) {
+	// This avoids an issue with module constructors overriding core functions
```

Comment:
Can we link to https://github.com/dagger/dagger/issues/9122 inline?

Reply:
Done